report/docker_run.sh

```diff
@@ -116,6 +116,18 @@ export ret_val=$?
 
 touch /experiment_ended
 
+# Generate and upload training data
+$PYTHON -m data_prep.parse_training_data \
+  --experiment-dir ${LOCAL_RESULTS_DIR:?} --save-dir 'training_data'
+$PYTHON -m data_prep.parse_training_data --group \
+  --experiment-dir ${LOCAL_RESULTS_DIR:?} --save-dir 'training_data'
+$PYTHON -m data_prep.parse_training_data --coverage \
+  --experiment-dir ${LOCAL_RESULTS_DIR:?} --save-dir 'training_data'
+$PYTHON -m data_prep.parse_training_data --coverage --group \
+  --experiment-dir ${LOCAL_RESULTS_DIR:?} --save-dir 'training_data'
+gsutil -q cp -r 'training_data' \
+  "gs://oss-fuzz-gcb-experiment-run-logs/Result-reports/${GCS_REPORT_DIR:?}"
```

Comment:
Think more about this, can we relocate this to https://github.com/google/oss-fuzz-gen/blob/main/report/upload_report.sh#L86?

This is because some of the latest full experiments have [termination problems](https://github.com/google/oss-fuzz-gen/issues/138), and this won't be executed when that happens.

We will need to make a few adjustments:
1. Remove the `training_data` directory in the bucket before uploading new ones. This is because the names of JSON files will be different. Hence, new ones cannot overwrite old ones.
2. Check if the local result dirs contain the necessary files/dirs. This can be done at the new line added in [8a9c71e](https://github.com/google/oss-fuzz-gen/pull/148/commits/8a9c71e0a7fd698dbeb879e647dddfc0a78b6f6f). The necessary files/dirs include `prompt.txt`, `raw_targets/`, `fixed_targets/`, and `status/`.

Reply:
This is [a sample way](https://github.com/google/oss-fuzz-gen/blob/main/report/web.py#L138) to validate and filter the directories.

Reply:
done